src/store/modules/brushModule.js

```diff
@@ -66,34 +57,30 @@ const mutations = {
       state.visibleSegmentations[enabledElementUID].push(true);
     }
   },
-  SET_ELEMENT_BRUSH_VISIBILITY: (enabledElementUID, segIndex, visible = true) => {
+  setBrushVisibilityForElement: (
+    enabledElementUID,
+    segIndex,
+    visible = true
+  ) => {
     if (!state.visibleSegmentations[enabledElementUID]) {
       state.imageBitmapCache[enabledElementUID] = [];
     }
 
     state.visibleSegmentations[enabledElementUID][segIndex] = visible;
   },
-  SET_ELEMENT_IMAGE_BITMAP_CACHE: (enabledElementUID, segIndex, imageBitmap) => {
+  setImageBitmapCacheForElement: (enabledElementUID, segIndex, imageBitmap) => {
     if (!state.imageBitmapCache[enabledElementUID]) {
       state.imageBitmapCache[enabledElementUID] = [];
     }
 
     state.imageBitmapCache[enabledElementUID][segIndex] = imageBitmap;
   },
-  CLEAR_ELEMENT_IMAGE_BITMAP_CACHE: (enabledElementUID) => {
+  clearImageBitmapCacheForElement: (enabledElementUID) => {
     state.imageBitmapCache[enabledElementUID] = [];
   }
-
 };
 
 const getters = {
-  draw: () => state.draw,
```

Comment:
I think we still need these api endpoints so that the parent app has easy accessibility to them?

Reply:
Think we resolved this in slack :).